build-support/scripts/up

```diff
@@ -37,3 +53,5 @@ docker run -d --name te2 --hostname te2 --rm \
     --net nuodb-net ${IMG_NAME} nuodocker \
     --api-server ad1:8888 start te \
     --db-name test --server-id ad1
+echo "Done!"
```

Comment:
Should we insert an invocation of [`nuocmd check database`](https://doc.nuodb.com/nuodb/latest/reference-information/command-line-tools/nuodb-command/nuocmd-reference/#check-database) here?